config/routes.rb

```diff
@@ -1,4 +1,15 @@
 Webplatform::Application.routes.draw do
+
+  get 'password_resets/create'
```

Comment:
@pg20 please remove lines 3-7 as you added route for resetting password in line 24